net/http/authz/store.go

```diff
@@ -9,28 +9,29 @@ import (
 	"chain/errors"
 )
 
+const keyPrefix = `/core/grant/`
```

Comment:
Ah, I see. Yeah, this is sort of analogous to hard-coding a specific SQL table name into the net/http/authz package, which isn't ideal. I'll add a TODO for now.